tests/clickhouseQuery_test.go

```diff
@@ -0,0 +1,22 @@
+package tests
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestUnmarshalJson(t *testing.T) {
+	jsonData := json.RawMessage(`{"multiplsier": 100}`)
```

Comment:
there is a typo. mulitpier.

trying to understand how the test was able to unmarshall to type ClickhouseQueryJobRaw

Reply:
it was not. Thanks for pointing out.